treeherder/etl/perf_data_adapters.py

```diff
@@ -101,8 +95,7 @@ def _calculate_test_data(job_id, result_set_id, push_timestamp,
             "job_id": job_id,
             "result_set_id": result_set_id,
             "push_timestamp": push_timestamp,
-            "total_replicates": r_len,
```

Comment:
We're not using it for anything right now, and leaving it out will let us simplify some things in the larger scale performance refactoring I'm doing. It will be easy to re-add it in the future if there is justification.